client/packages/system/src/Item/Components/PackVariantEditModal/PackVariantEditModal.tsx

```diff
@@ -94,16 +96,29 @@ export const PackVariantEditModal: FC<PackVariantEditModalProps> = ({
 }) => {
   const { Modal } = useDialog({ isOpen, onClose });
   const t = useTranslation('catalogue');
+  const { mutateAsync } = usePackVariantDelete();
   const { draft, onUpdate, onSave, isLoading } = useDraftPackVariant(
     itemId,
     packVariant,
     mode
   );
   const isInvalid = !draft.packSize && !draft.itemId;
+  const onDelete = async () => {
+    await mutateAsync(draft);
+    onClose();
+  };
+
+  const getConfirmation = useConfirmationModal({
```

Comment:
We have a pattern of modal on modal now... not sure if we want to change this...

Reply:
yeah, not great. but we can live with it